src/commands/wizard/lwcGenerationCommand.ts

```diff
@@ -9,18 +14,52 @@ export type QuickActionStatus = {
 };
 
 export type SObjectQuickActionStatus = {
+    error?: string;
     sobjects: {
         [name: string]: QuickActionStatus;
     };
 };
 
+export type GetSObjectsStatus = {
+    error?: string;
+    sobjects: string[];
+};
+
 export class LwcGenerationCommand {
     extensionUri: Uri;
 
     constructor(extensionUri: Uri) {
         this.extensionUri = extensionUri;
     }
 
+    static async getSObjectsFromLandingPage(): Promise<GetSObjectsStatus> {
+        return new Promise<GetSObjectsStatus>(async (resolve) => {
+            const staticResourcesPath = await UIUtils.getStaticResourcesDir();
```

Comment:
It looks good if this tool is specifically for landing page coming from static resources. But, landing page uem can also come from other sources, e.g., mobile builder, the default (packaged inside app).